interpreters/Wasm.mk

```diff
@@ -25,6 +25,10 @@ WASM_INITIAL_MEMORY ?= 65536
 STACKSIZE           ?= $(CONFIG_DEFAULT_TASK_STACKSIZE)
 PRIORITY            ?= SCHED_PRIORITY_DEFAULT
 
+ifeq ($(shell test $(WASM_INITIAL_MEMORY) -lt $(STACKSIZE); echo $$?),0)
+  WASM_INITIAL_MEMORY := $(shell expr $(STACKSIZE) + $(WASM_INITIAL_MEMORY))
+endif
```

Comment:
i guess it's simpler to stop specifying init memory. (at least for the default)

Reply:
Done, I changed INITMEMORY to a selectable configuration